bin/pdc_client

```diff
@@ -169,7 +169,7 @@ if __name__ == "__main__":
                 data[key] = ''
 
     try:
-        ssl_verify = options.ca_cert or not options.insecure
+        ssl_verify = options.ca_cert or options.insecure
```

Comment:
@ycheng-aa ssl_verify is a bit confused. i guess this variable means ca_cert_or_insecure, its value is None/pathtocert/True
is that correct? @bliuredhat